rightscale/resource_security_group_rule.go

```diff
@@ -115,67 +127,72 @@ func resourceSecurityGroupRule() *schema.Resource {
 				Elem:     &schema.Schema{Type: schema.TypeMap},
 				Computed: true,
 			},
+			"href": {
+				Type:        schema.TypeString,
+				Description: "href of security group rule",
```

Comment:
href, Href, HREF which one is it? :) (shouldn't be HREF).

Reply:
I'm doing 'href' unless its the start of a sentence.  Seem reasonable?